source/opt/dominator_analysis.cpp

```diff
@@ -55,6 +55,16 @@ bool DominatorAnalysisBase::Dominates(Instruction* a, Instruction* b) const {
     return tree_.Dominates(bb_a, bb_b);
   }
 
+  // We handle OpLabel instructions explicitly since they are not stored in the
+  // instruction list.
+  if (a->opcode() == SpvOpLabel) {
+    return true;
+  }
+
+  if (b->opcode() == SpvOpLabel) {
```

Comment:
Technically unnecessary since iterating will never find it.

Reply:
Fixed in d1c377e.